angular-todo/src/app/views/login/login.component.ts

```diff
@@ -8,7 +8,13 @@
  * of the MIT license. See the LICENSE file for details.
  */
 
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { environment } from '../../../environments/environment';
+import {
+  TokenManager, UserManager,
```

Comment:
It seems like some of this code could use some formatting. Can you try to format this code using VS Code?